java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -2136,6 +2174,20 @@ public void zrangestore_by_lex(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void zrangestore_same_slot(BaseClient client) {
```

Comment:
is it ok to keep it here? zrangestore has 3 test cases - with different range types

Reply:
cross slot and range type seem like separate arguments.  It doesn't seem necessary to write another set of 3 tests.